db/memtable.h

```diff
@@ -107,7 +107,8 @@ class MemTable {
                     const ImmutableOptions& ioptions,
                     const MutableCFOptions& mutable_cf_options,
                     WriteBufferManager* write_buffer_manager,
-                    SequenceNumber earliest_seq, uint32_t column_family_id);
+                    SequenceNumber earliest_seq, uint32_t column_family_id,
+                    const uint32_t* filter_bits = nullptr);
```

Comment:
Is there an advantage to making this be a pointer defaulting to nullptr vs a value defaulting to 0?

Reply:
The Google C++ style guide recommends not using default values for parameters.

Reply:
Following @anand1976 's recommendation,  I changed the way the filter bits are passed to the memtable, I am now updating the MutableCFOptions of the cfd directly.